saleor/graphql/core/mutations.py

```diff
@@ -854,11 +985,13 @@ class Meta:
         )
 
     @classmethod
-    def perform_mutation(cls, _root, info, **data):
-        file_data = info.context.FILES.get(data["file"])
+    def perform_mutation(  # type: ignore[override]
+        cls, _root, info: ResolveInfo, /, file
+    ):
+        file_data = info.context.FILES[file]
```

Comment:
Not familiar with that mutation, but from that snippet it looks like it could raises `KeyError` here (?)

Typing could be nice for `file_data` too, below it does `file_data.name` but what's the type?